### ISSUE TITLE
Sun-and-Earth-velocity-in-galaxy-astropy

### ISSUE THREAD

**Issue by KLFAMO** [KLFAMO/DMAnaliza#13]
In `earth_movement.py`:
- [x] convert polish description to English (just use some translator)
- [x] check and modify if necessary `sun_speed_astropy` function, we want the function to return vector of the speed of the Sun in galaxy in xyz coordinates (coordinates fixed with the Earth)
- [ ] - check and modify if necessary `earth_velocity_xyz_astropy` function, we want the function to return vector of the speed of the earth in galaxy in xyz coordinates (coordinates fixed with the Earth)
- [ ] create folder `doc` in the main project's folder and inside `doc` create file `earth_movement.md` (may be also latex with generated pdf) with description of the above calculations (brief description of coordinates and calculations/transformations)
- [ ] Tests: check if results are reasonable - compare absolute values with some values from literature (it should be around 230km/s), check if the speed vector of Earth rotates 360deg during one day.

In this task we want to use astropy as much as possible.

**Comment by jstrubingtardy** [comment]
Hi, I've been trying to make sense of the two astropy functions but there is still something I fundamentally don't understand. You're saying you would like the velocity of Earth in Earth-centred coordinates but that the value should be around 230 km/s. But this value is the speed of earth if you view it in the galactic frame, ie. not at all the speed you would observe from earth. I've been looking for ages but can't find any literature with values of the velocity of earth from an earth-centred frame, and i'm wondering if it even physically makes sense because I would think you need a frame outside of earth to be able to observe its velocity? 
So I am wondering if it wouldn't make more sense to get the velocity of earth in galactic frame or even try to get it in the comoving rest frame and then try to estimate which direction the earth is pointing at at any given moment to know which lab "goes through" the topological defect first. 
And if we need values of velocity for an earth-centred frame, I am really not sure where to get literature to confirm this, as I haven't found any yet. As I said, the value of 230 km/s is only valid for the galactic frame of reference.
Let me know what you think!

**Comment by KLFAMO** [comment]
Yes, you are absolutely right. The speed should be in galactic frame. But from the Earth point of view we have to take into account the rotation of the Earth (to know the direction).  We can get speed (vector) of the Earth in galaxy coordinates and then represent/convert this vector into the Earth's (xyz) coordinates. I hope it makes sense.
